Packs/Core/Triggers/Trigger_-_Suspicious_access_to_shadow_file.json

```diff
@@ -0,0 +1,18 @@
+{
+  "trigger_id": "41f9310c50c55b761fdc0aa5c48d6459",
+  "playbook_id": "Suspicious access to shadow file",
```

Comment:
because this playbook handles only 1 alert, lets update the playbook name and trigger to match the detection name